src/ripple/core/impl/Config.cpp

```diff
@@ -212,12 +216,12 @@ void Config::setup (std::string const& strConf, bool bQuiet)
 
     QUIET       = bQuiet;
 
-    strDbPath           = Helpers::getDatabaseDirName ();
-    strConfFile         = strConf.empty () ? Helpers::getConfigFileName () : strConf;
-
-    VALIDATORS_BASE     = Helpers::getValidatorsFileName ();
+    strDbPath = databaseDirName;
 
-    VALIDATORS_URI      = boost::str (boost::format ("/%s") % VALIDATORS_BASE);
+    if (!strConf.empty())
+        strConfFile = strConf;
+    else
+        strConfFile = configFileName;
```

Comment:
No, but `configFileName` is a `char const* const` and `strConf` is a `std::string` so I'd need to massage it a bit to work:

    strConfFile = strConf.empty() ? std::string(configFileName) : strConf;